### PR TITLE
Add copied files

### PR DESCRIPTION
**Pull request by Jonathan-kiprono**
In this pull request, I added 
1. Index.html
2. Stlyle.css
3. A babel folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      .container{
+  display: flex;
+}
+    </style>
```
Comment (Jonathan-kiprono) [inline index.html:11]: Please have all your CSS codes in the style.css file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <nav id="nav-bar">
+        <a class="nav-link" href="#upper">Features</a>
+        <a class="nav-link" href="#middle">How It Works</a>
+        <a class="nav-link" href="#lower">Pricing</a>
+      </nav>      
+    </header>
+    <video src="" id="video"></video>
+<div id="middle">
+  <section id=""></section>
+</div>
```
Comment (Jonathan-kiprono) [inline index.html:42]: Please add a valid id for the section .
Proper id naming helps in styling the css

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      .container{
+  display: flex;
+}
```
Comment (Jonathan-kiprono) [inline index.html:10]: Please include all your CSS lines f code in your style.css file.
Having your code base clean improves readability

**Review by Jonathan-kiprono — COMMENTED** [review]
Kindly check out the required changes.
Review them and submit once again.
cheers, Happy coding!

File: index.html (hunk 3 above)
Comment (binhussen) [reply index.html:10]: -[Optional] Kindly, use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.

File: index.html (hunk 2 above)
Comment (binhussen) [reply index.html:42]: -[Optional] Kindly, use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.

**Review by binhussen — APPROVED** [review]
Hi @Jonathan-kiprono , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All comments phrased  politely ✔️
- Comments include the reason why✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**